### PR TITLE
Don't obscure the "muted hero" area background w/ the page background

### PR DESCRIPTION
**Pull request by notlee**
Relates to:
e78f98f61b56ed5495ac3f79a6c690f1cb4f47e6

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-layout bundle size difference from 4.4.1 to 611cb14a73f34bcf608e8532307788122190511f
No bundle size differences found.

File: src/scss/_grid.scss [hunk 1]
```diff
@@ -5,6 +5,9 @@
 	html,
 	body {
 		margin: 0;
+	}
+
+	body {
```
Comment (JakeChampion) [inline src/scss/_grid.scss:10]: Did having the background-color apply on the html element cause an issue?

**Review by JakeChampion — APPROVED** [review]

File: src/scss/_grid.scss (hunk 1 above)
Comment (notlee) [reply src/scss/_grid.scss:10]: It does due to the negative z-index used on muted hero

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v4.5.0 :tada:

**Pull request merged**